_toc.rst

```diff
@@ -13,6 +13,7 @@
    cloud-backup
    cloud-database
    autoscale-heat-event-schedule-based
+   autoscale-stack-launch-type
```

Comment:
I think you have to add this to https://raw.githubusercontent.com/rackerlabs/rs-heat-docs/master/index.rst as well.